collectors/python.d.plugin/example/README.md

```diff
@@ -5,7 +5,10 @@ custom_edit_url: https://github.com/netdata/netdata/edit/master/collectors/pytho
 
 # Example
 
-An example python data collection module.
-You can use this example to help you [write a new Python module](/collectors/python.d.plugin/README.md#how-to-write-a-new-module).
+You can add custom data collectors using Python.
+
+Netdata provides an [example python data collection module](https://github.com/netdata/netdata/tree/master/collectors/python.d.plugin/example).
+
+If you want to write your own collector, read our [writing a new Python module](/collectors/python.d.plugin/README.md#how-to-write-a-new-module) tutorial.
```

Comment:
Is the content of the file published somewhere? e.g. on https://learn.netdata.cloud/docs/agent/collectors/python.d.plugin#how-to-write-a-new-module 
perhaps linking to the published page instead of the actual file view on the repo?

Reply:
Hi @siamaktavakoli 
The tool we use for publishing automatically converts these links to learn.netdata.cloud links. :)